helpers/internal.psm1

```diff
@@ -370,6 +370,30 @@ Function Get-DisqualifiedNetworksFromMapping {
     Return $Disqualified
 }
 
+Function Get-VDiskStatus {
+    param ( $LogFile )
+
+    $UnhealthyDisks = @()
+    Write-Host "Getting Virtual Disk Health..."
+    "Getting Virtual Disk Health..." | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+
+    Get-VirtualDisk | ForEach-Object {
+        if ($_.HealthStatus -eq 'Unhealthy') {
+            $UnhealthyDisks += $_.FriendlyName
+        } 
+    }
+    if ($UnhealthyDisks.Length -gt 0) { 
+        Write-Host "$(UnhealthyDisks) are unhealthy." 
+        "$(UnhealthyDisks) are unhealthy." | Out-File $LogFile -Append -Encoding utf8 -Width 2000

```

Comment:
This looks like incorrect syntax for the Subexpression operator:

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.1#subexpression-operator--

It either should be $($UnhealthyDisks) or just $UnhealthyDisks